executor/index_merge_reader.go

```diff
@@ -320,6 +308,28 @@ func (w *partialTableWorker) fetchHandles(ctx context.Context, fetchCh chan<- *l
 	}
 }
 
+func (w *partialTableWorker) handleHandlesFetcherPanic(ctx context.Context, resultCh chan<- *lookupTableTask, err error) func(r interface{}) {
+	return func(r interface{}) {
+		if r == nil {
+			return
+		}
+
+		buf := make([]byte, 4096)
+		stackSize := runtime.Stack(buf, false)
+		buf = buf[:stackSize]
```

Comment:
no need to record the call stack in this function, `WithRecovery()` will handle this. See the implementation of `WithRecovery()` for details.